src/allowed_args.cpp

```diff
@@ -230,6 +231,8 @@ static void addGeneralOptions(AllowedArgs &allowedArgs, HelpMessageMode mode)
             _("Execute command when the best block changes (%s in cmd is replaced by block hash)"))
         .addDebugArg("blocksonly", optionalBool,
             strprintf(_("Whether to operate in a blocks only mode (default: %u)"), DEFAULT_BLOCKSONLY))
+        .addArg("blockdbtype=<n>", requiredInt,
+            strprintf(_("Which method to store blocks on disk (default: %u) 1 = sequential files, 2 = leveldb, 3 = both")), DEFAULT_BLOCK_DB_MODE)
         .addArg("checkblocks=<n>", requiredInt,
```

Comment:
so if we're going to support both simultaneously, then what are the rules around a chain rollback or reorg?  which set of blocks are we expected to use?  Isn't this possibly going to cause confusion for the devs in the future; maybe we should just do one or the other but not both?  food for thought.